src/components/App/App.tsx

```diff
@@ -73,11 +74,24 @@ class App extends React.Component<AppProps, AppState> {
           <header className="App-header">
             <h1 className="header-text">Well<span className="header-text-2">News</span></h1>
           </header>
-          {this.state.userSentiment ? <Feed articles={this.state.articles} /> : <Form changeUserSentiment={this.changeUserSentiment}/>}
+          <Router>
+            <Switch>
+              <Route exact path="/">
+                <Form changeUserSentiment={this.changeUserSentiment}/>
+              </Route>
+              <Route path="/WellNewsFeed/">
```

Comment:
Would having a Redirect here avoid us having multiple Links for each of the buttons? I have never uses Redirect, but I am curious to see if this would be a good use case.

Reply:
Def something to be looked into! I'll let you know what I turn up.